cocotb/triggers.py

```diff
@@ -459,7 +459,7 @@ def retval(self):
         return self._coroutine.retval
 
     def prime(self, callback):
-        """FIXME: document"""
+        """Don't call this in your own code, this is for internal use by the scheduler."""
```

Comment:
I have a local branch with better docstrings than this, but haven't pushed it as an attept not to swamp @imphil with PRs

Reply:
Edit: gh-1019